app/assets/stylesheets/application.scss

```diff
@@ -147,3 +154,610 @@ ul > li > ul > li {
+  position: relative;
+  display: inline-block;
+  width: 32px;
+  top: -15px;
+  left: -30px;
+}
+
+.prompt .govuk-grid-column-one-quarter {
+  margin-right: -90px;
+}
+
+.prompt .govuk-grid-column-three-quarters {
+  padding: 15px 0;
+}
+
+@media (max-width: 40em) {
+  .prompt .govuk-grid-column-three-quarters {
+    padding-left: 15px;
+  }
+}
+
+@media (min-width: 40em) and (max-width: 61.5em) {
+  .prompt .govuk-grid-column-three-quarters {
+    padding-left: 40px;
+  }
+}
+
+.prompt .gem-c-govspeak {
+  padding-top: 20px;
+}
+
+@media (max-width: 40.0625em) {
+  .prompt .gem-c-govspeak {
+    padding-right: 20px;
+  }
+}
+
+@media (max-width: 40.0625em) {
+  .prompt .small-screen-padding-right {
+    padding-right: 20px;
+  }
+}
+
+.prompt.prompt-bg {
+  background-color: #d4f4f2;
+}
+
+.svg-inline--fa {
+  display: var(--fa-display, inline-block);
+  height: 1em;
+  overflow: visible;
+  vertical-align: -.125em;
+}
+
+.svg-inline--fa {
+  background-color: #28a197;
+  color: #ffffff;
+  padding: 20px !important;
+}
+
+.fa-2x {
+  font-size: 2em;
+}
+
+.gem-c-govspeak h2 {
+  margin-top: 45px;
+  margin-bottom: 0;
+}
+
+.gem-c-govspeak h2:first-child,
+.gem-c-govspeak h3:first-child,
+.gem-c-govspeak h4:first-child,
+.gem-c-govspeak p:first-child {
+  margin-top: 0;
+}
+
+.light-grey-background {
+  background-color: #f3f2f1;
+  padding: 30px;
+}
+
+.embed-container {
+  position: relative;
+  padding-bottom: 56.25%;
+  height: 0;
+  overflow: hidden;
+  max-width: 100%;
+}
+
+iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: none;
+}
+
+.white-background {
+  background-color: #ffffff;
+  border-left: none !important;
+  padding: 20px;
+}
+
+.js-enabled .gem-c-print-link {
+  display: block;
+}
+
+.gem-c-print-link__link,
+.gem-c-print-link__button {
+  background: url(/public/images/icon-print.png) no-repeat 10px 50%;
+  background-size: 16px 18px;
+  padding: 10px 10px 10px 36px;
+}
+
+.gem-c-print-link__button {
+  border: 1px solid #b1b4b6;
+  color: #1d70b8;
+  cursor: pointer;
+  margin: 0;
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 14px;
+  font-size: 0.875rem;
+  line-height: 1.1428571429;
+  margin-top: 0;
+  margin-bottom: 15px;
+}
+
+.dfe-content-page--header {
+  background: #ebf2f6;
+}
+
+.pb-pre-text {
+  color: $govuk-secondary-text-colour
+}
+
+.pb-outer {
+  width: 100%;
+  position: relative;
+  height: 10px;
+  background-color: #347ba9;
+}
+
+.pb-progress {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: $govuk-success-colour;
+  border-right: 3px #FFF solid;
+  height: 10px;
+}
+
+.module-section--item {
+  font-size: 1.1875rem;
+}
+
+.interrupt-panel {
+  width: 100%;
+  padding: 30px;
+  background: #1b68ab;
+}
+
+/*
+.dfe-width-container {
+  margin: 0 auto;
+}
+*/
+
+.card:hover .govuk-tag.govuk-tag--ey {
```

Comment:
We have dedicated SCSS files for card, govspeak etc. Would it be better to keep them all next to each other and introduce new files for individual elements to prevent this file from growing ever larger?

Reply:
possibly, i think i'd rather get all the design changes in and then look at refactoring the scss files.

Reply:
Whilst you have it all to hand it'd likely be less of a hassle to do it now IMHO. Would it be the very next PR for you or something for later down the line?

Reply:
I'd say once Rick is happy with everything in the progress bar series of stories (5, 3 completed) then we can look at refactoring.  You don't want to be refactoring whilst also trying to get the current code to work with rick's scss.